src/main/java/com/hubspot/jinjava/LegacyOverrides.java

```diff
@@ -0,0 +1,55 @@
+package com.hubspot.jinjava;
+
+/**
+ * This class allows Jinjava to be configured to override legacy behaviour.
+ * LegacyOverrides.NONE signifies that none of the legacy functionality will be overridden.
+ */
+public class LegacyOverrides {
+  public static final LegacyOverrides NONE = new LegacyOverrides.Builder().build();
+  private final boolean evaluateMapKeys;
+  private final boolean iterateOverMapKeys;
```

Comment:
`evaluateMapKeys` and `iterateOverMapKeys` seem to be the most fitting so far as they are config options that allow for either preserving legacy functionality, or matching Jinja2 functionality